src/selectors/entities/preferences.js

```diff
@@ -120,14 +120,18 @@ const getThemePreference = createSelector(
     }
 );
 
+const getDefaultThemeName = createSelector(getConfig, (config) => config.DefaultTheme || 'default');
+
 export const getTheme = createShallowSelector(
     getThemePreference,
-    (themePreference) => {
+    getDefaultThemeName,
+    (themePreference, defaultThemeName) => {
         let theme;
+        const defaultTheme = Preferences.THEMES[defaultThemeName];
```

Comment:
We need to make sure this theme exists or else the app will crash on an invalid DefaultTheme. If you actually change the selector to `getDefaultTheme` like I mentioned above, you could do the check inside the selector so that it's safe if it gets used elsewhere